files/zh-cn/web/api/document/startviewtransition/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Document.startViewTransition()"
```

Comment:
Question, why don't you follow the pattern of the [en-US document](https://github.com/mdn/content/blob/4785cd825d3e74ffdd11740d19473cb3ae63832c/files/en-us/web/api/document/startviewtransition/index.md?plain=1#L2) which is the newest style we use.

Reply:
I followed the format of other Chinese documents, like [Document.cookie](https://developer.mozilla.org/zh-CN/docs/Web/API/Document/cookie) and [Document.open()](https://developer.mozilla.org/zh-CN/docs/Web/API/Document/open).

If need to update according to the new style, please wait.